core/storage/classifier/gae_models_test.py

```diff
@@ -178,6 +178,23 @@ def test_create_multi_jobs(self):
         self.assertEqual(training_job2.data_schema_version, 1)
 
 
+    def test_id_collisions_exception(self):
```

Comment:
Move this a line up. As a general convention, functions are preceeded by a single newline.